### ISSUE TITLE
Filters get temporarily disabled when using HD Mode 7 without supersampling

### ISSUE THREAD

**Issue by MisaelK** [byuu/bsnes#180]
Whenever a scene with Mode 7 appears and resolution is higher than 240p, filters get temporarily disabled, unless supersampling is enabled.

In short:
Scale 240p: filters work.
Scale >240p, supersampling off: filters don't work.
Scale >240p, supersampling on: filters work.

https://streamable.com/c3qgu

https://streamable.com/enq4p

Tested with bsnes v111r8 and v111r9, with Direct3D and OpenGL on Windows 7 x64. Specs: i5-3550, 16GB, GTX 1050 Ti.

**Comment by byuu** [comment]
Unfortunately that one's by design. Imagine trying to apply HQ2x to HD mode 7 at 3840x2160. I don't think there exists a PC in the world that is going to render 8K mode 7.

I believe hunterk modified the xBRZ shaders to be able to handle 480p and 720p HD mode 7, but that's really about the best you're going to get.

**Comment by MisaelK** [comment]
I was under the impression that filters got applied last on the video pipeline, like shaders are.

Another thing I just realized after playing a bit with HD Mode 7, filters, and shaders, is that the internal resolution increases when increasing HD Mode 7's scale. For some reason I thought that HD Mode 7 was somehow a "pre-step" on the renderer pipeline, and so the window resolution (that is, setting size as 1x, 2x, 3x, 4x) was always the real resolution.

I understood this as I applied CRT-Geom, because the scanlines change size when increasing HD Mode 7's scale.

Anyway, thanks for answering!
